src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -162,40 +169,55 @@ private function getIdentifier(Request $request, MapEntity $options, string $nam
         }
 
         if ($request->attributes->has($name)) {
-            return $request->attributes->get($name);
+            return $request->attributes->get($name) ?? ($options->stripNull ? false : null);
```

Comment:
@stof @jderusse would this added logic make sense to you, in relation with #47242?

Reply:
works for me (with a test case 😉 )

Reply:
test case updated